src/core/lender/assessor.sol

```diff
@@ -58,19 +61,25 @@ contract Assessor is DSNote,DSMath {
         else revert();
     }
 
+    function calcAssetValue(address tranche) public returns(uint) {
+        uint trancheReserve = TrancheLike(tranche).balance();
+        uint poolValue = pile.debt();
+        if (tranche == junior) {
+            return calcJuniorAssetValue(poolValue, trancheReserve, seniorDebt());
+        }
+        return calcSeniorAssetValue(poolValue, trancheReserve, SeniorTrancheLike(tranche).debt(), juniorReserve());
```

Comment:
you could just used seniorDebt() instead of SeniorTrancheLike(tranche).debt()

Reply:
we could but I think it is a bit cheaper not doing it